bin/stashcp

```diff
@@ -97,6 +97,14 @@ def doStashCpSingle(sourceFile, destination, cache, debug=False):
     payload['start2']=start2
     payload['end2']=end2
     
+    with open(".job.ad") as job_file:
```

Comment:
- Might want to use a separate function for this; monitoring not directly related to this function
- Need to catch exceptions.
- Preferentially look at `$_CONDOR_JOB_AD` env var over the hardcoded filename.